### PR TITLE
Modify openid scope claims

### PR DESCRIPTION
**Pull request by Thisara-Welmilla**
Remove all user attributes for 'openid' scope.

### CODE REVIEW

**Review by IsuraD — APPROVED** [review]

**Comment by emswbandara** [comment]
@Thisara-Welmilla let's link the git issue with the PR

File: components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java [hunk 1]
```diff
@@ -2016,7 +2016,14 @@ public static OauthTokenIssuer getOAuthTokenIssuerForOAuthApp(OAuthAppDO appDO)
...
         }
-        return (claimConfig.toString().split(","));
+
+        String[] claim;
+        if (claimConfig.length() > 0) {
+            claim = claimConfig.toString().split(",");
+        } else {
+            claim = new String[0];
+        }
+        return (claim);
```
Comment (emswbandara) [inline components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java:2026]: shall we remove the redundant brackets here

File: features/org.wso2.carbon.identity.oauth.server.feature/resources/oidc-scope-config.xml [hunk 2]
```diff
@@ -17,9 +17,7 @@
 -->
 <Scopes>
     <Scope id="openid" displayName="Open ID" description="Indicate the use of OpenID Connect protocol.">
-        <Claim>
-            sub,email,email_verified,name,family_name,given_name,middle_name,nickname,preferred_username,upn,groups,profile,picture,website,gender,birthdate,zoneinfo,locale,updated_at,phone_number,phone_number_verified,address,street_address,country,formatted,postal_code,locality,region,roles
-        </Claim>
+        <Claim></Claim>
```
Comment (emswbandara) [inline features/org.wso2.carbon.identity.oauth.server.feature/resources/oidc-scope-config.xml:20]: can't we remove this element ?

File: components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java (hunk 1 above)
Comment (Thisara-Welmilla) [reply components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java]: Addressed with commit 3e36e5f40f310a3901eed2b66fc54932223cf789

File: features/org.wso2.carbon.identity.oauth.server.feature/resources/oidc-scope-config.xml (hunk 2 above)
Comment (Thisara-Welmilla) [reply features/org.wso2.carbon.identity.oauth.server.feature/resources/oidc-scope-config.xml]: Addressed with commit 3e36e5f40f310a3901eed2b66fc54932223cf789

**Review by emswbandara — APPROVED** [review]

**Pull request merged**